man/istr.Rd

```diff
@@ -47,11 +47,11 @@ needs to produce its products.The input products are, for example, resources,
 packaging materials, energy and enabling services (such as tractor use on farm)
 to produce the product.
 
-If an input product belongs to a sector in which reduction targets are over 70\%,
-it belongs to a high-risk sector and is classified as 'high sector transition
+If an input product belongs to a sector in which reduction targets are strictly
+over 66.66\%, it belongs to a high-risk sector and is classified as 'high sector transition
```

Comment:
@lindadelacombaz I don't know why there was a "\" in the first place... Do you know? If not, please get rid of the "\" in front of %-signs in the whole article. 
Also, after double checking how <= / < are set, please eventually update the signs as above.